bundles/org.openhab.binding.pjlinkdevice/src/main/java/org/openhab/binding/pjlinkdevice/internal/PJLinkDeviceHandler.java

```diff
@@ -194,6 +251,27 @@ public void handleCommand(ChannelUID channelUID, Command command) {
     }
   }
 
+  private @Nullable String getChannelTypeId(ChannelUID channelUID) throws ConfigurationException {
+    @Nullable
+    Channel channel = thing.getChannel(channelUID);
```

Comment:
Put the @Nullable on the same line as the variable delcaration

Reply:
I'll only have access to my dev environment at the end of next week. I've had a quick look before and I found that this @nullable is not required at all. I'll remove it as soon as I can access my dev env again.